app/handlers/card_created_handler.rb

```diff
@@ -0,0 +1,12 @@
+class CardCreatedHandler < BasicEventHandler
+  delegate :project_card, to: :event
+  delegate :column_id, to: :project_card
+
+  def execute!
+    Card.create(project_card.with_params(user: sender, column: column).to_hash)
```

Comment:
Спробуй в app/models/application_record.rb додати:
```ruby
def create(params)
  super(params.to_hash)
end
```
тоді
```suggestion
    Card.create(project_card.with_params(user: sender, column: column))
```

Reply:
Так не працює
`Error performing HandleEventJob (Job ID: 828a9c08-9bc8-40a4-97f4-8033a27a010e) from Async(default) in 10.96ms: ArgumentError (When assigning attributes, you must pass a hash as an argument, CardStruct passed.)`